commit 3664aec05930381d8904d165eca07eec2eae6089 (refs/changes/34/2634/2)
Author: Angela Fong <angela.fong@joyent.com>
Date:   2017-09-22T15:48:35+00:00 (2 years, 1 month ago)
    
    ADMINUI-2386 Want Guideline on Swap Space for Packages
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/www/js/tpl/packages-form.hbs b/www/js/tpl/packages-form.hbs
index 1beb9832..256e1d35 100644
--- a/www/js/tpl/packages-form.hbs
+++ b/www/js/tpl/packages-form.hbs
@@ -142,6 +142,7 @@
                 name="max_swap" value="{{max_swap}}" min="128" placeholder="swap">
             <span class="input-group-addon">MB</span>
           </div>
+          <p class="help-block">Must be at least twice the size of memory</p>
         </div>
       </div>
 
